testing/rundeck/APKBUILD

```diff
@@ -0,0 +1,86 @@
+# Contributor: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+# Maintainer: Jean-Baptiste Guerraz <jbguerraz@gmail.com>
+pkgname=rundeck
+pkgver=2.6.9
+pkgrel=0
+pkgdesc="Job scheduler and runbook automation. Enable self-service access to existing scripts and tools."
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+url="http://rundeck.org"
+arch="noarch"
+license="ASL 2.0"
+depends="openjdk8 bash ncurses5-libs"
```

Comment:
Does it really need complete JDK? IMO it needs only headless JRE, so `openjdk8-jre-base` would be more appropriate as runtime dependency (and `openjdk8` for build).